h/static/scripts/directive/sidebar-tutorial.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+// @ngInject
+function controller($http, session, features) {
+  /*jshint validthis:true */
+  var vm = this;
+
+  vm.showSidebarTutorial = function () {
+    return (features.flagEnabled('sidebar_tutorial') &&
+            session.state.show_sidebar_tutorial);
+  };
+
+  vm.dismiss = function () {
+    $http({method: 'POST', url: '/sidebar_tutorial/dismiss'});
```

Comment:
I'm not entirely sure about this. The session is an `ngResource` already, so I think it would make more sense to add a method on the session which will result in a request to the server to modify the server state.

I'd rather see this implemented as an `action` on the `session` resource than as an HTTP call hiding in a directive.